docs/Fields.md

```diff
@@ -545,11 +580,11 @@ import { SelectField } from 'react-admin';
 
 ### Properties
 
-| Prop              | Required | Type                          | Default | Description                                                                                                                                  |
-| ----------------- | -------- | ----------------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
-| `optionText`      | Optional | `string | function | Element` | 'name'  | Name of the field to use to display the matching choice, or function returning that field name, or a React element to render for that choice |
-| `optionValue`     | Optional | `string`                      | 'id'    | Name of the field to compare to the value to find the matching choice                                                                        |
-| `translateChoice` | Optional | `Boolean`                     | true    | Whether or not the choice text should be translated                                                                                          |
+| Prop              | Required | Type                                         | Default | Description                                                                                                                                  |
```

Comment:
Please refrain from reformatting tables like that - it's a never ending effort that has no real added value IMO.

Reply:
Same